haystack/nodes/ranker/lost_in_the_middle.py

```diff
@@ -0,0 +1,126 @@
+from typing import Optional, Union, List
+import logging
+
+from haystack.schema import Document
+from haystack.nodes.ranker.base import BaseRanker
+
+logger = logging.getLogger(__name__)
+
+
+class LostInTheMiddleRanker(BaseRanker):
```

Comment:
I actually like LITM name because it is unique and well descriptive and associative with that now "famous" paper. So I'd keep it if there are no objections...

Reply:
I like the LITM name in a sense that it is a clear reference to the paper. I don't think that the users would be specifically searching for this type of ranking without understanding the background for it?

Anyway, I tried to think about possible alternatives, but no good ideas so far. They're coming up pretty bad (IrrelevantMiddleRanker, TopBottomRanker, IrrelevantDocsInTheMiddleRanker...)

Reply:
Right, I see what you are saying. And I see how this is different compared to our other rankers. I think my preference would be to have the user control this through just the `top_k` alone for consistency with our other rankers, but I can see their usefulness. 

One thing to point out though is that even with these parameters it does not guarantee that a future node like the PromptNode won't still further truncate the documents so this is still some possibility for error here.